doc/modules/ensemble.rst

```diff
@@ -128,15 +128,15 @@ Random Forests
 --------------
 
 In random forests (see :class:`RandomForestClassifier` and
-:class:`RandomForestRegressor` classes), each tree in the ensemble is
-built from a sample drawn with replacement (i.e., a bootstrap sample)
-from the training set. In addition, when splitting a node during the
-construction of the tree, the split that is chosen is no longer the
-best split among all features. Instead, the split that is picked is the
-best split among a random subset of the features. As a result of this
-randomness, the bias of the forest usually slightly increases (with
-respect to the bias of a single non-random tree) but, due to averaging,
-its variance also decreases, usually more than compensating for the
+:class:`RandomForestRegressor` classes), each tree in the ensemble is built
+from a sample drawn with replacement (i.e., a bootstrap sample) from the
+training set. In addition, when splitting a node during the construction of the
+tree, the split that is chosen is no longer the best split among all features,
+unless you leave the default parameter ``max_features=’auto’``. Instead, the
```

Comment:
this is not true?
``max_features=’auto’`` = ``max_features=’None'`` =  ``max_features=n_features``

Reply:
For random forest **regressor**, yes
`max_features=’auto’` = `max_features=’None'` = `max_features=n_features`
https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/tree/tree.py#L221

But I noted that for random forest classifier
(`max_features=’auto’` = `max_features=’sqrt(n_features)'`) != (`max_features=’None'`=`max_features=n_features`)

Reply:
But you're using unless here, I think it's misleading. Try to rephrase your words.

Reply:
Done